compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -204,6 +204,7 @@
     INVALID_FUNCTION_POINTER_INVOCATION("invalid.function.pointer.invocation"),
     AMBIGUOUS_TYPES("ambiguous.type"),
 
+    FUNCTION_POINTER_REST_ARG_ARRAY_PASSED("function.pointer.rest.arg"),
```

Comment:
Shall we rename this? This doesn't self explain what is this error/warning?

Reply:
Renamed to `ARRAY_PASSED_TO_REST_ARG`